tests/release/release.go

```diff
@@ -74,40 +73,41 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 
 	var _ = Describe("Creation of the 'Happy path' resources", func() {
 
-		It("Create an Snapshot in dev namespace.", func() {
+		It("creates a Snapshot in dev namespace.", func(ctx SpecContext) {
 			_, err := framework.ReleaseController.CreateSnapshot(snapshotName, devNamespace, applicationName, snapshotComponents)
 			Expect(err).NotTo(HaveOccurred())
-		})
+			// We add the namespace creation timeout as this is the first test so must also take into account the code in BefforeAll
```

Comment:
should just be one f in BeforeAll instead of BefforeAll. Can you change the spacing so it aligns with line 80?

Reply:
Fixed it, thanks